interface/main/finder/dynamic_finder.php

```diff
@@ -162,33 +170,39 @@ function persistCriteria(el, e) {
 
 </script>
 <?php
-//to determine and set the page to open in the desired state - expanded or centered, any selection the user makes will
-//become the user-specific default for that page. collectAndOrganizeExpandSetting() takes a single indexed array as an
-//argument, containing one or more elements, the name of the current file is the first element, if there are linked
-//files they should be listed thereafter, please add _xpd suffix to the file name
-$arr_files_php = array("dynamic_finder_xpd");
-$current_state = collectAndOrganizeExpandSetting($arr_files_php);
-require_once("$srcdir/expand_contract_inc.php");
-?>
-<script>
-<?php require_once("$include_root/expand_contract_js.php");//jQuery to provide expand/contract icon toggle if page is expandable ?>
-</script>
-
+    $arrOeUiSettings = array(
+    'heading_title' => xl('Patient Finder'),
+    'include_patient_name' => false,
+    'expandable' => true,
+    'expandable_files' => array('dynamic_finder_xpd'),//all file names need suffix _xpd
+    'action' => "search",//conceal, reveal, search, reset, link or back
+    'action_title' => "",//only for action link, leave empty for conceal, reveal, search
+    'action_href' => "",//only for actions - reset, link or back
+    'show_help_icon' => false,
+    'help_file_name' => ""
+    );
+    $oemr_ui = new OemrUI($arrOeUiSettings);
+    ?>
 </head>
 <body class="body_top">
-    <div class="<?php echo $container;?> expandable">
-        <div class="row">
-            <div class="col-sm-10">
-                <h2>
-                <?php echo xlt('Patient Finder') ?> <i id="exp_cont_icon" class="fa <?php echo attr($expand_icon_class);?> oe-superscript-small expand_contract"
-                title="<?php echo attr($expand_title); ?>" aria-hidden="true"></i> <i id="show_hide" class="fa fa-search-plus fa-2x small" title="<?php echo xla('Click to show advanced search'); ?>"></i>
-                </h2>
+    <div id="container_div" class="<?php echo attr($oemr_ui->oeContainer()); ?>">
+         <div class="row">
+            <div class="col-sm-12">
+                <div class="page-header">
+                    <?php echo $oemr_ui->pageHeading() . "\r\n"; ?>
+                </div>
             </div>
-            <div class="col-sm-2">
+        </div>
+        <div class="row">
+            <div class="col-sm-12">
                 <?php if (acl_check('patients', 'demo', '', array('write','addonly'))) { ?>
-                    <button id="create_patient_btn" onclick="top.restoreSession();top.RTop.location = '<?php echo $web_root ?>/interface/new/new.php'"><?php echo xlt('Create Patient'); ?></button>
+                    <button id="create_patient_btn1" class="btn btn-default btn-add" onclick="top.restoreSession();top.RTop.location = '/openemr-master/interface/new/new.php'">Add New Patient</button>
```

Comment:
Guessing the above line change is not meant to be there ( hard-codes path, doesn't translate)

Reply:
Sorry missed in cleanup